trl/trainer/dpo_trainer.py

```diff
@@ -389,74 +388,22 @@ class DPOTrainer(Trainer):
 
     _tag_names = ["trl", "dpo"]
 
-    @_deprecate_arguments(
-        version="1.0.0",
```

Comment:
Out of curiosity, is the reason to remove these deprecated args due to the fact we've had this warning for several releases?

I wonder if it's better to pin the `version` to 2-3 future versions to give people a concrete warning?

Reply:
For context, these arguments have been deprecated in #1554, in [v0.9.3](https://github.com/huggingface/trl/releases/tag/v0.9.3).

Sounds good, let's wait a bit.